ee/ucp/admin/configure/restrict-services-to-worker-nodes.md

```diff
@@ -44,34 +41,27 @@ $ kubectl get nodes <ucpmanager> -o json | jq -r '.spec.taints | .[]'
 }
 ```
 
-> Note: Workloads deployed by an Administrator in the `kube-system` namespace do
-> not follow these scheduling constraints. If an Administrator deploys a
-> workload in the `kube-system` namespace, a toleration is applied to bypass
-> this taint, and the workload is scheduled on all node types.
+> **Note**
+> 
+> Workloads deployed by an administrator in the `kube-system` namespace do not follow these scheduling constraints. If an administrator deploys a workload in the `kube-system` namespace, a toleration is applied to bypass this taint, which then schedules the workload on all node types. 
 
-### Allow Administrators to Schedule on Manager / DTR Nodes
+### Allow administrators to schedule workloads on manager / DTR nodes
 
-To allow Administrators to deploy workloads accross all nodes types, an
-Administrator can tick the "Allow administrators to deploy containers on UCP
-managers or nodes running DTR" box in the UCP web interface. 
+To allow administrators to deploy workloads across all nodes types, select the **Allow administrators to deploy containers on UCP managers or nodes running DTR** check box in the UCP UI. 
 
 ![](../../images/restrict-services-to-worker-nodes-2.png){: .with-border}
 
-For all new workloads deployed by Administrators after this box has been
-ticked, UCP will apply a toleration to your workloads to allow the pods to be
-scheduled on all node types.
+When this option is enabled, for all new workloads deployed by administrators, UCP will apply a toleration to workloads to allow the pods to be scheduled on all node types. 
```

Comment:
Suggest removing the first comma (after 'enabled').

Also, suggest changing "to allow" to "that allows".

Reply:
I believe these suggestions would alter the intent.

Reply:
Perhaps one of these works better...?

"When this option is enabled, UCP will apply a toleration to all new workloads deployed by administers which allows the pods to be scheduled on all node types."

-- or --

"When this option is enabled, for all new workloads deployed by administrators UCP will apply a toleration that allows the pods to be scheduled on all node types."